tests/RobotFramework/tests/cumulocity/software.robot

```diff
@@ -47,6 +47,7 @@ Custom Setup
     Device Should Exist                      ${DEVICE_SN}
     Set Test Variable    $DEVICE_SN
     Should Have MQTT Messages    tedge/health/tedge-mapper-c8y
+    Execute Command    sudo start-http-server.sh
```

Comment:
sudo is necessary because I don't want to assume the commands are being run as root. But in the container environment it is a no-op if the user is already root. It is more about making sure the commands also work in other environments (e.g. using ssh, and local adapters).